### ISSUE TITLE
Recordings which are stopped, then resumed only ingest the first part

### ISSUE THREAD

**Issue by gregorydlogan** [opencast/zoom-ingest#66]
Zoom allows recordings to be paused and then resumed.  In the backend this appears to be accomplished by stopping the recording, then starting a new one for each section.  Unfortunately, these sections are not then concatenated at the end of the overall recording.  Instead these are presented as separate files, in the same way that the different recording views are presented.  The current code makes the incorrect assumption that each view represents the *full* recording, and so grabs the first match and ingests this.

Action items:
- Find all of the files
- _Check that they are in order, since it's unclear if Zoom does this_
- Ingest them all
- Trigger a workflow which concatenates them prior to normal processing

Workaround: Encourage your users to not pause the recording, or to start a completely new session.

**Comment by oas777** [comment]
I'm not sure, but I think this problem was not caused by pausing the recording but by stopping the recording. Do you want me to test this with "pause recording"?

**Comment by gregorydlogan** [comment]
Yes, please.  Hopefully this isn't a *second* case that's broken as well...

**Comment by oas777** [comment]
Pausing the recording leaves you with one recording (four video files obviously), so it's a problem of "stopping" the recording. Which makes sense somehow.
